src/sql/changes/2025-07-15-add-last-login-and-analyst-to-users.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE users
+  ADD COLUMN IF NOT EXISTS last_login_date timestamptz,
```

Comment:
```suggestion
  ADD COLUMN IF NOT EXISTS last_login_date TIMESTAMP WITH TIME ZONE DEFAULT NULL,
```
for consistency with user_tables.sql.